### PR TITLE
improve unit test in order to reduce execution time.

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/sdc/installator/impl/InstallatorPuppetImpl.java [hunk 1]
```diff
@@ -209,8 +207,7 @@ private void callPuppetMaster(VM vm, String vdc, ProductRelease product, String
         HttpResponse response;
 
         log.info("Calling puppetWrapper " + action);
-        log.info("connecting to puppetURL: " + "puppetURL: " + puppetUrl + "v2/node/" + vm.getHostname() +
-                "/" + action);
+        log.info("connecting to puppetURL: " + "puppetURL: " + puppetUrl + "v2/node/" + vm.getHostname() + "/" + action);
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/sdc/installator/impl/InstallatorPuppetImpl.java]: I have now 123 characters in this line...

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java [hunk 2]
```diff
@@ -206,8 +209,7 @@ public void testIsNodeDeployed() throws OpenStackException, CanNotCallPuppetExce
     }
 
     @Test
-    public void testLoadNode() throws OpenStackException, CanNotCallPuppetException, IOException, 
-    InstallatorException {
+    public void testLoadNode() throws OpenStackException, CanNotCallPuppetException, IOException, InstallatorException {
```
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java]: gt 120 chars.

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java [hunk 3]
```diff
@@ -330,55 +332,54 @@ public void testgetPuppetDBEndpointException() throws OpenStackException, CanNot
...
     }
-    
+
     @Test
-    public void testValidatorData() throws InvalidInstallProductRequestException, ClientProtocolException, 
-           IOException{
-        
-        VM host=mock(VM.class);
-        
+    public void testValidatorData() throws InvalidInstallProductRequestException, ClientProtocolException, IOException {
```
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java]: gt 120 chars.

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java (hunk 2 above)
Comment (jesuspg) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java]: with my IDE: just before limit
screen shot 2015-02-25 at 14 56 20

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java (hunk 3 above)
Comment (jesuspg) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java]: just before limit

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java (hunk 2 above)
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java]: ok in that case NTC, I see here 123 chars.

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java (hunk 3 above)
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorPuppetTest.java]: NTC

File: core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorChefTest.java [hunk 4]
```diff
@@ -150,11 +148,12 @@ public void setup() throws CanNotCallChefException, EntityNotFoundException, IOE
 
         propertiesProvider = mock(SystemPropertiesProvider.class);
 
-        installator = new InstallatorChefImpl();
+        installator = spy(new InstallatorChefImpl());
         installator.setRecipeNamingGenerator(recipeNamingGenerator);
         installator.setChefNodeDao(chefNodeDao);
         installator.setPropertiesProvider(propertiesProvider);
         installator.setSdcClientUtils(sdcClientUtils);
+        doNothing().when(installator).sleep(10000);
```
Comment (flopezag) [inline core/src/test/java/com/telefonica/euro_iaas/sdc/installator/InstallatorChefTest.java]: Magic number

**Comment by flopezag** [comment]
LGTM

**Pull request merged**